persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -122,6 +125,21 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
     case GetState =>
       sender() ! state
 
+    case GetStatePaginated(page) =>
+      val tsErrState = state.tsParsingErrors
+      val tsErrorCount = if (tsErrState.isEmpty) 0 else 1
+
+      val (tsErrorsReturn, offset) =
+        if (tsErrState.nonEmpty && page == 1) (tsErrState, 1)
+        else (Seq(), 0)
+
+      val totalLarErrors = state.larParsingErrors.size
+      val from = Math.min(totalLarErrors, (page - 1) * 20)
```

Comment:
I agree--configurable page size would be nice. I'm currently looking into ways to make all of this code more reusable (though that might be easier when I've implemented pagination on more endpoints). I'll see what I can do.